Dockerfile

```diff
@@ -28,7 +40,6 @@ RUN curl -o ./snyk-alpine https://static.snyk.io/cli/latest/snyk-alpine && \
     mv snyk-alpine /usr/local/bin/snyk && \
     chmod +x /usr/local/bin/snyk
 
-
```

Comment:
restore this newline since it it wasn't an area with any edits

Reply:
Fixing